main.go

```diff
@@ -118,6 +120,11 @@ func main() {
 		}
 		managerWatchCache = cache.MultiNamespacedCacheBuilder(namespaceList)
 	}
+	ctx := context.TODO()
+	if !webhookDisabled {
+		// we need to add indexers to KafkaTopics so that the KafkaTopic admission webhooks could work
+		managerWatchCache = k8sutil.AddKafkaTopicIndexers(ctx, managerWatchCache)
```

Comment:
Would it worth adding the indexers always regardless webhooks are enabled or disabled?

Reply:
I wasn't sure. Since we don't the use the fieldselectors if the webhooks are disabled, I did not see it necessary. We can enable it regardless.